pkg/config/xds/bootstrap/config.go

```diff
@@ -84,13 +84,13 @@ func (b *BootstrapParamsConfig) Validate() error {
 	return nil
 }
 
-func DefaultBootstrapParamsConfig() *BootstrapParamsConfig {
+func buildDefaultBootstrapParamsConfig(adminAccessLogPath string) *BootstrapParamsConfig {
```

Comment:
This seems unnecessarily complicated just to swizzle the name of the null device. Can you just do 

```Go
nullDeviceName := "/dev/null"

if runtime.GOOS == "windows" {
   nullDeviceName = "NUL"
}

Reply:
done